code/__DEFINES/jamming_defines.dm

```diff
@@ -0,0 +1,14 @@
+
+//Radio jammer levels
+//They will jam any signals that have the same or lower protection value
+#define RAIDO_JAMMER_ABDUCTOR_LEVEL 1	//Power of the abductor radio jammer
+#define RADIO_JAMMER_TRAITOR_LEVEL 3	//Power of the traitor radio jammer
```

Comment:
Any defines that are used across multiple files tend to go in __DEFINES because in byond the order in which you specify defines affects where you can use them. A defined defined in one file cannot be used in any file that comes alphabetically before the file that the define is in.

Reply:
No, it should be flags, in a var on the component, instead of an arbitrary numbering system where if anything is bigger it's allowed (e.g. the byond invisibility system, which sucks)

Reply:
Ah, I see what you mean now. I don't think its necessary however, the only current use-case is for blocking cameras for abductor jammers and if new jammers get introduced then they will most likely either be weaker or stronger. I think that if a use case for flags is needed in the future as oppossed to a simple comparison then it can be implemented by whoever needs it.

I'd rather keep it as numbers unless strictly necessary due to the number of jamming levels not being a clear constant and the need for them not being present (I don't see a case where it will be present anytime soon).

Reply:
Flags would be fine and work, but I don't think they would be more effective than just regular integers in this use-case and I'd rather not put in extra effort to code something that doesn't give any gain